packages/opentelemetry-exporter-collector/src/platform/browser/util.ts

```diff
@@ -55,8 +55,11 @@ export function sendWithXhr(
 ) {
   const xhr = new XMLHttpRequest();
   xhr.open('POST', url);
-  xhr.setRequestHeader('Accept', 'application/json');
-  xhr.setRequestHeader('Content-Type', 'application/json');
+
+  if (!Object.keys(headers).includes('Accept'))
```

Comment:
instead of doing this whole check you can remove it and then
```
const defaultHeaders = {
    'Accept': 'application/json',
    'Content-Type': 'application/json',
};

Object.entries({
    ...defaultHeaders,
    ...headers,
  }).forEach(([k, v]) => {
    xhr.setRequestHeader(k, v);
  });
```

Will be much easier to understand what is happening

Reply:
ok